tfhe/src/integer/server_key/radix_parallel/tests_unsigned/test_scalar_comparison.rs

```diff
@@ -398,6 +398,38 @@ fn integer_unchecked_scalar_comparisons_edge(param: ClassicPBSParameters) {
     }
 }
 
+// Tests whether equality holds for greater/less or equal than, but fails for greater/lesser than.
+fn integer_comparisons_for_empty_blocks(param: ClassicPBSParameters) {
+    let (cks, sks) = KEY_CACHE.get_from_params(param, IntegerKeyKind::Radix);
+
+    let scalar = 0 as u64;
```

Comment:
scalar should be picked at random, and in the assert compare the decrypted value with the clear comparison result e.g. `scalar < 0`

Reply:
Makes sense! In some of the assert the statements are trivially true, so I kept the booleans there.